.buildkite/hooks/pre-command

```diff
@@ -58,12 +58,6 @@ if [[ "$BUILDKITE_STEP_KEY" == *"integration-tests"* ]]; then
   export API_KEY_TOKEN=$(vault kv get -field apiKey ${CI_ESS_PATH})
   echo ${API_KEY_TOKEN} > ./apiKey
   export TEST_INTEG_AUTH_ESS_APIKEY_FILE=$(realpath ./apiKey)
-
-  # BK analytics
-  echo "--- Prepare BK test analytics token :vault:"
```

Comment:
The hooks are intended to inject secrets into the pipelines; why are we changing this?